versions.sh

```diff
@@ -3,6 +3,11 @@ set -Eeuo pipefail
 
 # defaultDebianSuite gets auto-declared below
 declare -A debianSuites=(
+# the libc created by gcc is too old for a newer Debian
+# $ apt --version
+# apt: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found
+	[10]="bullseye"
+	[9]="bullseye"
```

Comment:
```suggestion
	[10]='bullseye'
	[9]='bullseye'
```

(I think I'm going to apply this post-merge so we don't have to wait for CI again :upside_down_face:)